app/gateway-oss/2.3.x/upgrading.md

```diff
@@ -77,58 +78,55 @@ repository contains [openresty-build-tools](https://github.com/Kong/kong-build-t
 which allows you to build OpenResty with the necessary patches
 and modules easily.
 
-For Go support, you also need to build both your plugins and
-the [Kong go-pluginserver](https://github.com/kong/go-pluginserver).
-The documentation includes detailed [instructions on how to build
-the plugin server and your plugins](https://docs.konghq.com/2.2.x/go/).
+There is a new way to deploy Go using Plugin Servers. You can read more about it the [Go section of the docs](https://docs.konghq.com/2.3.x/go).
 
-#### 2. Template Changes
+### Template changes
 
 There are **Changes in the Nginx configuration file**, between kong 2.0.0,
-2.1.0 and 2.2.0.
+2.1.0, 2.2.0 and 2.3.0.
 
 To view the configuration changes between versions, clone the
 [Kong repository](https://github.com/kong/kong) and run `git diff`
 on the configuration templates, using `-w` for greater readability.
 
-Here's how to see the differences between 2.0.0 and 2.2.0:
+Here's how to see the differences between 2.0.0 and 2.3.0:
 
 ```
 git clone https://github.com/kong/kong
 cd kong
-git diff -w 2.0.0 2.2.0 kong/templates/nginx_kong*.lua
+git diff -w 2.0.0 2.3.0 kong/templates/nginx_kong*.lua
 ```
 
 To produce a patch file, use the following command:
 
 ```
-git diff 2.0.0 2.2.0 kong/templates/nginx_kong*.lua > kong_config_changes.diff
+git diff 2.0.0 2.3.0 kong/templates/nginx_kong*.lua > kong_config_changes.diff
 ```
 
-#### 3. Suggested Upgrade Path
+### Suggested upgrade path
 
-##### Upgrade from `0.x` to `2.2.0`
+#### Upgrade from `0.x` to `2.3.0`
 
-The lowest version that Kong 2.2.0 supports migrating from is 1.0.0.
+The lowest version that Kong 2.3.0 supports migrating from is 1.0.0.
 If you are migrating from a version lower than 0.14.1, you need to
 migrate to 0.14.1 first. Then, once you are migrating from 0.14.1,
 please migrate to 1.5.0 first.
 
 The steps for upgrading from 0.14.1 to 1.5.0 are the same as upgrading
 from 0.14.1 to Kong 1.0. Please follow the steps described in the
-"Migration Steps from 0.14" in the [Suggested Upgrade Path for Kong
-1.0](#kong-1-0-upgrade-path), with the addition of the `kong
-migrations migrate-apis` command, which you can use to migrate legacy
-`apis` configurations.
+"Migration Steps from 0.14" in the 
+[Suggested Upgrade Path for Kong 1.0](#kong-1-0-upgrade-path), 
+with the addition of the `kong migrations migrate-apis` command, 
+which you can use to migrate legacy `apis` configurations.
 
 Once you migrated to 1.5.0, you can follow the instructions in the section
-below to migrate to 2.2.0.
+below to migrate to 2.3.0.
 
-##### Upgrade from `1.0.0` - `2.1.0` to `2.2.0`
+#### Upgrade from `1.0.0` - `2.2.0` to `2.3.0`
 
 **Postgres**
 
-Kong 2.2.0 supports a no-downtime migration model. This means that while the
+Kong 2.3.0 supports a no-downtime migration model. This means that while the
 migration is ongoing, you will have two Kong clusters running, sharing the
 same database. (This is sometimes called the Blue/Green migration model.)
```

Comment:
I rewrote this section for EE (blue/green etc.) and it is a future opportunity for an include.